index.html

```diff
@@ -13,9 +13,20 @@
 </head>
 
 <body>
+    <h1>Welcome To Confession Page</h1>
     <div id="root"></div>
+    <div class="container inputarea">
+      <fieldset>
+      <legend>Enter details for confessions</legend>
+      <h1>Name of the person you want to confess</h1>
+      <input class="card" type="text" name="name" placeholder="Name" autocomplete="off">
+      <h1>Write your confession here</h1>
+      <textarea class="card" name="name" maxlength="200" placeholder="Enter confess in 200 max words........."></textarea>
```

Comment:
![Input text](https://user-images.githubusercontent.com/44135316/101656055-48f5a500-3a68-11eb-9b59-e1ed69f481c1.png)
Make sure textarea and input name field, have the same border margin, they look different.

Reply:
okayy okayy iwill correct it.